### PR TITLE
Use ParamHandling's `positive_definite`

### LINKED ISSUE #35: Improve parametrisation in examples
The classification notebook seems to not have very robust fitting. Could this be due to the parametrisation of the covariance of q(u)? (See  https://github.com/JuliaGaussianProcesses/SparseGPs.jl/pull/32#r694577357)

### PR DESCRIPTION
**Pull request by rossviljoen**
Should close #35 @st--

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # SparseGPs
 
-[![CI](https://github.com/rossviljoen/SparseGPs.jl/actions/workflows/CI.yml/badge.svg)](https://github.com/rossviljoen/SparseGPs.jl/actions/workflows/CI.yml)
-[![Codecov](https://codecov.io/gh/rossviljoen/SparseGPs.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/rossviljoen/SparseGPs.jl)
+[![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://JuliaGaussianProcesses.github.io/SparseGPs.jl/dev)
```
Comment (st--) [inline README.md:3]:
```suggestion
[![Docs](https://img.shields.io/badge/docs-dev-blue.svg)](https://JuliaGaussianProcesses.github.io/SparseGPs.jl/dev)
```
?

**Review by st-- — APPROVED** [review]
Seems to be optimising fine : ) do you want to try using a more "interesting" sample for the ground truth now?

**Comment by rossviljoen** [comment]
I already reduced the length scale to make it slightly more interesting - was there something else you had in mind?

**Comment by st--** [comment]
> I already reduced the length scale to make it slightly more interesting - was there something else you had in mind?

no, just remembered you saying more complicated functions didn't work due to optimisation issues. let's merge it as is, we can always improve it more later on : )

**Pull request merged**